chapters/dashboard/connect-jinaD.md

```diff
@@ -0,0 +1,48 @@
+#  Connect Dashboard to JinaD
+
+Before using the Dashboard with Jina, you'll need to connect it to JinaD first. <br/>
```

Comment:
maybe explain why we need JinaD

Reply:
I would put a link into the JinaD section. Otherwise, we have redundant information that becomes hard to update